src/main/java/io/buildpacks/example/sample/PeerReview.java

```diff
@@ -1,7 +1,5 @@
-// PeerReview.java
-
```

Comment:
2. Testing: The values used in the `main` method for testing are appropriate and cover different scenarios like positive, negative, and mix of positive and negative numbers.